manifests/j/jqlang/jq/1.6/jqlang.jq.installer.yaml

```diff
@@ -1,14 +1,14 @@
-# Created using wingetcreate 1.1.2.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+# Created with YamlCreate.ps1 v2.2.9 $debug=AUSU.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
 
 PackageIdentifier: jqlang.jq
-PackageVersion: 1.6
+PackageVersion: "1.6"
+InstallerType: portable
+Commands:
```

Comment:
This directive is mislocated. It should got under "Installers" directive according to the schema: https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.4.0/installer.md